cairo/ethereum/cancun/fork.cairo

```diff
@@ -473,6 +544,7 @@ func process_transaction{
     let total_gas_used = gas_used - gas_refund;
 
     // Refund gas to sender
+    let state = env.value.state;
```

Comment:
No need for this one, it is already line 523 above
It was done in https://github.com/kkrt-labs/keth/pull/670 

Line on main 
https://github.com/kkrt-labs/keth/blob/f0d50ada22e651d2106f11248c4e08dcf77205ab/cairo/ethereum/cancun/fork.cairo#L452